server/src/test/scala/org/scassandra/server/cors/CorsSupportTest.scala

```diff
@@ -0,0 +1,58 @@
+package org.scassandra.server.cors
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest._
+import org.scassandra.server.ScassandraConfig
+import org.scassandra.server.priming.cors.CorsSupport
+import spray.http._
+import spray.routing.HttpService
+import spray.testkit.ScalatestRouteTest
+
+
+class CorsSupportTest extends FlatSpec
+  with ScalatestRouteTest
+  with HttpService
+  with LazyLogging
+  with CorsSupport
+  with Matchers {
+  def actorRefFactory = system
+
+  val testRoute = path("test") {
```

Comment:
Thanks, yes, I though about adding them, but decided to way for your feedback first.
I’ll add them during the weekend.


> On 31 May 2016, at 13:29, Andrew Tolbert <notifications@github.com> wrote:
> 
> In server/src/test/scala/org/scassandra/server/cors/CorsSupportTest.scala <https://github.com/scassandra/scassandra-server/pull/162#discussion_r65171676>:
> 
> > +import org.scassandra.server.ScassandraConfig
> > +import org.scassandra.server.priming.cors.CorsSupport
> > +import spray.http._
> > +import spray.routing.HttpService
> > +import spray.testkit.ScalatestRouteTest
> > +
> > +
> > +class CorsSupportTest extends FlatSpec
> > +  with ScalatestRouteTest
> > +  with HttpService
> > +  with LazyLogging
> > +  with CorsSupport
> > +  with Matchers {
> > +  def actorRefFactory = system
> > +
> > +  val testRoute = path("test") {
> Very nice tests! They helped me understand CorsSupport better. My only suggestion here would be to also add a test to validate the case where ScassandraConfig.corsEnabled is false and ensure the rejected message is propagated.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/scassandra/scassandra-server/pull/162/files/25f99748f09f8a6290c5b72e805bde6764026c8e#r65171676>, or mute the thread <https://github.com/notifications/unsubscribe/AEcH5-HOQWtloIpQe16glJz4dzZDkGM7ks5qHCmQgaJpZM4Io98s>.
>